windows_containers/understanding-windows-container-workloads.adoc

```diff
@@ -17,7 +17,7 @@ The following Windows Server operating systems are supported for {product-title}
 
 * Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019
 
-For more information, see Microsoft's documentation on link:https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19[Windows Server channels].
+For more information, see Microsoft's documentation on link:!https://docs.microsoft.com/en-us/windows-server/get-started-19/servicing-channels-19[Windows Server channels].
```

Comment:
fyi @codyhoag

Reply:
@jboxman this would be a comparable replacement: https://docs.microsoft.com/en-us/windows-server/get-started/editions-comparison-windows-server-2019